src/bulk_insert.c

```diff
@@ -106,8 +106,8 @@ RedisModuleString** _Bulk_Insert_Read_Labeled_Node_Attributes(RedisModuleCtx *ct
 
     for(int i = 0; i < attribute_count; i++) {
         size_t attribute_len;
-        char *attribute_val = strdup((char*)RedisModule_StringPtrLen(*argv++, &attribute_len));
```

Comment:
`SIValue_FromString` will create its own copy, so no need to dup here. This also brings it into conformance with line 133 below.